src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -12,7 +12,6 @@
 
 import java.io.IOException;
 import java.util.List;
-import java.util.Map;
```

Comment:
I would recommend removing this import if you are not going to use it 

https://medium.datadriveninvestor.com/java-why-you-remove-unused-imports-7a3327a0b877